eos/effects/usemissiles.py

```diff
@@ -4,7 +4,27 @@
 # Modules from group: Missile Launcher Heavy (12 of 12)
 # Modules from group: Missile Launcher Rocket (15 of 15)
 # Modules named like: Launcher (151 of 151)
-type = 'active'
-def handler(fit, module, context):
+type = 'active', "projected"
+
+
+def handler(fit, src, context):
     # Set reload time to 10 seconds
-    module.reloadTime = 10000
+    src.reloadTime = 10000
+
+    if "projected" in context:
+        if src.item.group.name == unicode("Missile Launcher Bomb"):
```

Comment:
It's stored as unicode, yes, but it isn't unicode. Everything in that string could be ASCII.

My main point is that it's inconsistent (not that pyfa is consistent with much). In most (all?) other effect files we use a simple strings when matching groups, skills, etc, because there isn't any reason to use unicode. And if we are gonna use unicode, `u""` convention would be preferred.